File: Heat/ViewModels/PlantServices/CreatePlantServiceViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.PlantServices
    Public Class CreatePlantServiceViewModel

        Property PlantID As Integer

        <Display(name:="Impianto di riferimento")> _
        Property PlantDescription As String

        <Display(name:="Data dell'ultima manutenzione eseguita")> _
        Property PreviousServiceDate As Nullable(Of DateTime)

        <Display(name:="Periodicità della manutenzione")> _
        Property Periodicity As Periodicity

        <Display(name:="Scadenza legale della manutenzione")> _
        Property LegalExpirationDate As DateTime

        <Display(name:="Data programmata della prossima manutenzione")> _
        Property PlannedServiceDate As DateTime

    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/PlantServices/CreatePlantServiceViewModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.PlantServices
{
    public class CreatePlantServiceViewModel
	{

		public int PlantID { get; set; }

		[Display(Name = "Impianto di riferimento")]
		public string PlantDescription { get; set; }

		[Display(Name = "Data dell'ultima manutenzione eseguita")]
		public Nullable<DateTime> PreviousServiceDate { get; set; }

		[Display(Name = "Periodicità della manutenzione")]
		public Periodicity Periodicity { get; set; }

		[Display(Name = "Scadenza legale della manutenzione")]
		public DateTime LegalExpirationDate { get; set; }

		[Display(Name = "Data programmata della prossima manutenzione")]
		public DateTime PlannedServiceDate { get; set; }

	}
}

